### PR TITLE
feat: add xml support to cyclonedx handlers

### PR DESCRIPTION
**Pull request by cpendery**
## 📝 Description
Adds xml support for the CycloneDX guesser, processor, and ingestor.

Closes https://github.com/guacsec/guac/issues/102

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -82,10 +82,12 @@ jobs:
...
-          go-version: '1.18'
+          go-version: "1.18"
       - name: Install formatter
         run: go install golang.org/x/tools/cmd/goimports@latest
       - name: Check format
         run: make fmt
       - name: golangci-lint
         uses: golangci/golangci-lint-action@537aa1903e5d359d0b27dbc19ddd22c5087f3fbc # tag=v3.2.0
+        with:
+          args: --timeout 5m
```
Comment (cpendery) [inline .github/workflows/ci.yaml:93]: Runtime for this pr was `1.17`, so this should be more than enough time

**Comment by lumjjb** [comment]
hi @cpendery sorry for the late review on this - do you mind rebasing please! Thanks so much!

**Comment by lumjjb** [comment]
Hi @cpendery ! hope all is going well! Just checking in!

**Comment by lumjjb** [comment]
👋 @cpendery

**Comment by cpendery** [comment]
@lumjjb Let me know if the changes don't work. Sorry again about the delay

**Review by pxp928 — APPROVED** [review]
LGTM! Thanks @cpendery

**Pull request merged**